_im-plugin/ism/policies.md

```diff
@@ -438,6 +439,84 @@ Parameter | Description | Type | Required
 ]
 ```
 
+### rollup
+
+[Index rollup](https://opensearch.org/docs/latest/im-plugin/index-rollups/index/) lets you periodically reduce data granularity by rolling up old data into summarized indices.
+
+Only a single rollup job can be created with this method. The source index for the rollup job is inferred from the index the policy it is attached to.
```

Comment:
do you want to show them the syntax for the request too? Similar to the other ISM API sections:
https://opensearch.org/docs/latest/im-plugin/ism/api/#create-policy

would it be PUT or POST (?)
PUT _plugins/_ism/rollup